schemas/allowed-amounts/README.md

```diff
@@ -73,4 +73,7 @@ Negotiated rates for items and services can come from a variety of billing code
 | All Patient Diagnosis Related Groups | AP-DRG | |
 | All Patient Refined Diagnosis Related Groups | APR-DRG | [AHRQ documentation](https://www.hcup-us.ahrq.gov/db/nation/nis/APR-DRGsV20MethodologyOverviewandBibliography.pdf) |
 | Ambulatory Payment Classifications | APC | [APC background information](https://www.acep.org/administration/reimbursement/reimbursement-faqs/apc-ambulatory-payment-classifications-faq/#question0) |
-
+| Local Processing  | LOCAL | |
```

Comment:
We haven't added this table yet to the allowed amounts file - can you please remove this from the PR and I'll update this file once the in-network table is up-to-date.